docs-content/installing.html.md.erb

```diff
@@ -49,141 +59,99 @@ When you successfully configure AppDynamics Nozzle on PCF, a default application
+ ip:"192.168.16.21" tags:<key:"source_id" value:"uaa" >
+ valueMetric:<name:"audit_service.principal_not_found_count" value:0 unit:"gauge" >
+```
+to report the metric the filter entry will be
 
-1. Click the **Restart** button to restart the app.
+```json
+{
+    "uaa": ["audit_service.principal_not_found_count"]
+}
+```
 
-    ![AppDynamics KPI Dashboard Restart button](./images/pcf-dashboard-restart-app-ui.png)
+### <a id='Overriding Defaults'></a> (Optional) Overriding Defaults
 
-## <a id='overide-nozzle-ops-manager'></a> Advanced Configuration
+<p class="note"> For 4.4.x controllers it is mandatory to override the default Application, Tier, Tier ID and Node values.
 
-To use AppDynamics Controllers v4.4.x, you must first create an instance of an application before deploying the AppD Platform Monitoring tile.
-You can do this using either the Apps Manager or from the Cloud Foundry Command Line Interface (cf CLI). The app name argument is optional, so that they match with your existing application.
-If an app name is not specified, default names for the AppDynamics Nozzle, `appd-nozzle`, the Nozzle Tier,
-`appd-nozzle-tier` and Nozzle Node, `appd-nozzle-node` will be generated.
-It is often easier to start with a default name and rename the app later.
-Record the application name, tier name and node name and tier id. Use these recorded values to fill in the Advanced Configuration for the app.
+![Settings tab for AppDynamics Default Overrides](./images/4.png)
 
-For AppDynamics Controllers version 4.5.x, when you create a new app, you'll need to know the API endpoint for your Cloud Foundry instance.
-This is the URL of the Cloud Controller in your Pivotal Application Service (PAS) instance, also known as the target URL.
-See [Identifying the API Endpoint for your PAS Instance](https://docs.pivotal.io/pivotalcf/2-3/opsguide/api-endpoint.html) for more information.
-You must select a `Go SDK` tier, but the tier id is not necessary for Advanced Configuration.
+* **UAA Token Endpoint**: UAA token end point of the cloudfoundry. This can be found by doing `$ cf curl /v2/info | grep token_endpoint`
+* **Nozzle Application Name**: Enter a name to identify the Nozzle Application on the Controller, such as **_test-app-nozzle_**.
+* **Nozzle Tier Name**: Enter a name for the Nozzle Tier that corresponds to the Nozzle Application Name, such as **_test-app-tier_**. This MUST be a Go SDK Tier.
+* **Nozzle Node Name**: Enter a name for the Nozzle Node that corresponds to the Nozzle Application Name, such as **_test-app-node_**.
+* **Nozzle Tier ID**: Enter the component ID of the GoLang SDK tier.
 
-**To rename the Nozzle Application, Tier and Node names**
+#### <a id='44x Controllers'></a> 4.4.x Controllers
+For 4.4.x controller, please create an applicaiton on the AppDynamics Controller and create a `Go SDK` tier associated with that application and record the tier-id.
```

Comment:
For 4.4.x controller, ~please~ create an application on the AppDynamics Controller. Create a `Go SDK` tier associated with that application and record the tier-id.